src/components/NavItem/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { Link, useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { Paths } from '../../paths';
+
+interface NavItemProps {
+  path: Paths;
+  defaultIcon: React.ReactNode;
+  selectedIcon: React.ReactNode;
+  text: string;
+}
+
+const Icon = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  height: 35px;
```

Comment:
can we use `rem` for height and width to be more responsive to the root font size?

Reply:
The icons will still grow in response to the font size setting, if that's what you are trying to achieve - or are you referring to some other problem? 👀 The icon's sizing is determined by the FontAwesome component itself, and this is just a container - so perhaps I should rename this to `IconWrapper` or something along the line to avoid the confusion.
<img width="680" alt="Screenshot 2022-05-10 at 16 04 29" src="https://user-images.githubusercontent.com/32060873/167540422-60f79fde-e2d1-4710-b8e0-4033afebbecb.png">

On a related note, since this is not the first time you've given the feedback, are you perhaps under the impression that 'using rem is best practice and using px is bad'? 🙂 There seems to be a lot of discussion / confusion around this topic so happy to go into details offline.

Reply:
haha I just tested locally and you are right - the icons are responsive with the browser font size so apologies for the wrong comment! that was my concern and I should have tested before I comment here. yeah it will be great to have a session on rem/px - I don't think using px is a bad practice. I wanted to make sure it's responsive to the root font size.